ktor-features/ktor-metrics-micrometer/jvm/test/io/ktor/metrics/micrometer/MicrometerMetricsTests.kt

```diff
@@ -259,15 +266,16 @@ class MicrometerMetricsTests {
             }
 
             assertNull(throwableCaughtInEngine)
-            assertTrue(noHandlerHandledReqeust)
+            assertTrue(noHandlerHandledRequest)
+            testRegistry.assertEqualRequestTimers()
```

Comment:
Not sure if we need it for every test. I think it's better to add an extra test that verifies new properties have an old value by default and that's it.